libs/NfcCoreLib/lib/LibNfc/phHciNfc_Pipe.c

```diff
@@ -1194,6 +1193,16 @@ phHciNfc_ReceiveOpenPipeNotifyCmd(void *pContext,NFCSTATUS wStatus, void *pInfo)
                             PH_LOG_HCI_CRIT_STR("No need to launch sequence");
                         }
                     }
+                    else if (pReceivedParams->bPipeId == pHciContext->aGetHciSessionId[PHHCI_ESE_APDU_PIPE_STORAGE_INDEX])
```

Comment:
`if (pReceivedParams->bPipeId == pHciContext->aGetHciSessionId[PHHCI_ESE_APDU_PIPE_STORAGE_INDEX])` -
This check is redundant, the statement is always true according to comparison on line 1174

Reply:
That's true a simple else was enough.
Fixing...